src/routes/backpack/base.php

```diff
@@ -50,4 +50,8 @@ function () {
         Route::post('edit-account-info', 'MyAccountController@postAccountInfoForm')->name('backpack.account.info.store');
         Route::post('change-password', 'MyAccountController@postChangePasswordForm')->name('backpack.account.password');
     }
+
+    // This defines the fallback route when no matching route is found, is responsible for showing the 404 page.
+    // NOTE: this should be the last registered route in this file.
+    Route::fallback('AdminController@backpackHttpNotFoundView');
```

Comment:
What if we use a closure for the `Route:fallback()` instead of adding a Controller method to it?

Reply:
Did it, much cleaner 👌
And will be even more with PHP8 arrow functions, in a couple years when PHP8 will be the minimum requirement 😅

Reply:
@tabacitu just a quick note about what I told you yesterday; Pedro had approved this PR a long time ago, today I pushed a new commit and his approval was not invalidated by default 🤷‍♂️
I'm not going to assign him, since it's a small change, I'll let you decide ✌